_includes/setup/path-win.md

```diff
@@ -0,0 +1,24 @@
+### Update your path
+
+You can update your PATH variable for the current session only at the command line,
+as shown in [Clone the Flutter repo](./#clone-the-repo). You'll probably want to 
```

Comment:
For windows, we don't describe a way to only set it for the current session anywhere (`setx` is permanent, I think).

As it stands, I don't find the information in this section very useful. We already describe how to set a permanent user variable with `setx` elsewhere, so why do we need to describe how to set it via the UI here? Also, why are we concerned about system variables if a user variable is all we really need?